test/union-props/output.json

```diff
@@ -13,8 +13,8 @@
 						"types": [
 							{ "type": "UndefinedNode" },
 							{ "type": "LiteralNode", "value": "\"standard\"" },
-							{ "type": "LiteralNode", "value": "\"outlined\"" },
-							{ "type": "LiteralNode", "value": "\"filled\"" }
+							{ "type": "LiteralNode", "value": "\"filled\"" },
+							{ "type": "LiteralNode", "value": "\"outlined\"" }
```

Comment:
This updated even before the switch to TS@3.8. Unsure what's going on here

Reply:
Sorting change that probably wasn't commited, but didn't fail the test